Add symfony config docs (#810)
getsentry/sentry-docs: src/collections/_documentation/platforms/php/symfony.md

        request: 1
        subrequest: 1
        console: 1
```
### DSN
The DSN option is the only required one: it sets the Sentry DSN, and so reports all events to the related project. If it's
left empty, it actually disables Sentry reporting. Since the bundle is enabled in all environments, it's recommended to 
disable it in the `test` and `dev` environments.

### Options
All the possibile configurations under the `options` key map directly to the correspondent options from the base SDK;
you can read more about those in the [Unified API configuration docs]({% link _documentation/error-reporting/configuration/index.md %}),
and on the [PHP specific SDK docs]({% link _documentation/platforms/php/index.md %}#php-specific-options).

Below you can find additional documentation that is specific to the bundle usage, or information about the sensible default
values that are used in some cases.

#### before_breadcrumb and before_send
The `before_breadcrumb` and `before_send` options both accept a `callable`; thus, you cannot provide it directly through
 a YAML file; the bundle accepts a service reference (starting with `@`), which you can build in your DIC container.

#### environment

> The `environment` option defaults to the same environment of your Symfony application (i.e. `dev`, `test`, `prod`...).

Reviewer: ```suggestion
The `environment` option defaults to the same environment of your Symfony application (For example,`dev`, `test`, `prod`...).
```
```suggestion
The environment option defaults to the same environment of your Symfony application (For example, dev, test, prod, etc...).
```

#### in_app_exclude
The `in_app_exclude` option is used to mark files as non belonging to your app's source code in events' stack traces.
In this bundle it has two default values: 
 * `%kernel.cache_dir%`, to exclude Symfony's cache dir
 * `%kernel.project_dir%/vendor`, to exclude Composer's dependencies

#### project_root
The `project_root` options defaults to `%kernel.project_dir%`.

### Listener priorities

> The `listener_priorities` options are used to change the priority at which the bundle's listener are called. They all default

Reviewer: ```suggestion
The `listener_priorities` options are used to change the priority at which the bundle's listener is called. They all default
```
